libs/ui-watt/src/lib/components/expandable-card/watt-expandable-card.component.scss

```diff
@@ -0,0 +1,55 @@
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+@use "@energinet-datahub/watt/utils" as watt;
+
+.watt-expandable-card.mat-expansion-panel {
+  border-radius: 4px !important;
+
+  & .mat-expansion-panel-header {
+    @include watt.typography-watt-headline-4;
+    height: var(--watt-space-xl);
+    padding: 0 var(--watt-space-m);
+    user-select: none;
+
+    &:hover:not([aria-disabled="true"]) {
+      background: var(--watt-color-neutral-white);
+    }
+  }
+
+  & .mat-expansion-indicator {
+    margin-top: -6px; // align vertically with text
+    transform-origin: center 15px; // modify origin to match visual center
+
+    &::after {
+      border-color: var(--watt-color-primary);
+    }
+  }
+
+  & .mat-expansion-panel-body {
+    border-top: 1px solid var(--watt-color-neutral-grey-300);
+    padding: 0 var(--watt-space-m);
+  }
+
+  & watt-expandable-card-badge {
+    @include watt.space-inset-squish-xs;
+    @include watt.typography-watt-text-m-highlighted;
+    margin-right: var(--watt-space-m);
+    border-radius: 4px;
+    color: var(--watt-color-primary);
+    background: var(--watt-color-state-info-light);
```

Comment:
Question: Should the badge always be `--watt-color-state-info-light`, which brings me back to my other comment if the badge should just be a normal badge inside of a title styled a bit differently?